staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -41,12 +41,22 @@ var (
 		},
 		[]string{"transformation_type"},
 	)
-	transformerFailuresTotal = prometheus.NewCounterVec(
+	transformerOperationsTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "transformation_operations_total",
+			Help:      "Total number of transformations.",
+		},
+		[]string{"transformation_type", "status"},
+	)
+	// Deprecated, use transformerOperationsTotal instead.
+	deprecatedTransformerFailuresTotal = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: namespace,
 			Subsystem: subsystem,
 			Name:      "transformation_failures_total",
-			Help:      "Total number of failed transformation operations.",
+			Help:      "Deprecated, use transformerOperationsTotal instead. Total number of failed transformation operations.",
```

Comment:
The deprecated description could follow this [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/0031-kubernetes-metrics-overhaul.md#deprecation-plan) , for example: https://github.com/kubernetes/kubernetes/blob/e8daa90ca791efddaa846b2b42baceb158d90586/pkg/kubelet/dockershim/metrics/metrics.go#L124

Reply:
What @danielqsj commented is the correct deprecation procedure. We should do it consistently in this PR.